src/io-methods/default.js

```diff
@@ -1,30 +1,26 @@
 "use strict";
 const _ = require("lodash");
-const fs = require("./filesystem");
+const filesystem = require("./filesystem");
```

Comment:
I found the naming `fs` to be confusing, since `fs` is a builtin module in NodeJS. So I renamed this.